test/DynamoCoreWpfTests/PublishPackageViewModelTests.cs

```diff
@@ -47,7 +47,7 @@ public void SetsErrorState()
             //open a dyf file and modify it
             string packagedirectory = Path.Combine(TestDirectory, "pkgs");
             var packages = Directory.EnumerateDirectories(packagedirectory);
-            var first = Path.GetFullPath(packages.First());
+            var first = Path.GetFullPath(packages.FirstOrDefault(x => Path.GetFileName(x).Equals("Custom Rounding")));
```

Comment:
These tests specifically check for the "Custom Rounding" test package that earlier happened to be the first package in the list. This order changed in this PR after I added new test packages, which is why I'm specifically searching for the "Custom Rounding" package from now on.